endpoint.go

```diff
@@ -1000,5 +1000,17 @@ func (c *controller) cleanupLocalEndpoints() {
 				log.Warnf("Could not delete local endpoint %s during endpoint cleanup: %v", ep.name, err)
 			}
 		}
+
+		epl, err = n.getEndpointsFromStore()
+		if err != nil {
+			log.Warnf("Could not get list of endpoints in network %s for count update: %v", n.name, err)
+			continue
+		}
+
+		epCnt := n.getEpCnt().EndpointCnt()
+		if epCnt != uint64(len(epl)) {
+			log.Warnf("Inconsistent endpoint_cnt for network %s. Expected=%d, Actual=%d", n.name, len(epl), epCnt)
```

Comment:
Given we are fixing the endpoint count if incorrect, we should not throw a warning, rather an info or a debug.
Not a blocker, we can take care of this later.